test/currentUser.test.ts

```diff
@@ -0,0 +1,92 @@
+import asyncFn from '@async-fn/jest'
+import {renderHook, act} from '@testing-library/react-hooks'
+import {createCurrentUserHook} from '../src/currentUser'
+
+// eslint-disable-next-line prefer-promise-reject-errors
+global.fetch = () => Promise.reject('Forgot to mock fetch')
+
+describe('createCurrentUserHook', () => {
```

Comment:
I know it _probably_ doesn't matter, and this is not a bulletproof solution, but it would help my zen if we'd try to restore the original fetch, eg something like:

```ts
const originalFetch = global.fetch

describe('createCurrentUserHook', () => {
  this.beforeEach(() => {
    global.fetch = Promise.reject(new Error('Forgot to mock fetch'))
  })

  this.afterEach(() => {
    global.fetch = originalFetch
  }) 
```